maps_dashboards/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "./target",
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    // adding global typings
+    "noImplicitAny": false,
```

Comment:
This is not good practice, why is this necessary?

Reply:
will remove it

Reply:
removed

Reply:
Oh man, thanks for the callout. should remove it there as well. Thanks.